pkg/server/endpoints/authorized_entryfetcher_test.go

```diff
@@ -652,3 +685,75 @@ func TestUpdateAttestedNodesCacheMissedStartupEvents(t *testing.T) {
 	require.Equal(t, entry.EntryId, entries[0].Id)
 	require.Equal(t, entry.SpiffeId, idutil.RequireIDProtoString(entries[0].SpiffeId))
 }
+
+// AgentsByIDCacheCount
+func agentsByIDMetric(val float32) fakemetrics.MetricItem {
+	return fakemetrics.MetricItem{
+		Type:   fakemetrics.SetGaugeType,
+		Key:    []string{telemetry.Node, telemetry.AgentsByIDCache, telemetry.Count},
+		Val:    val,
+		Labels: nil}
+}
+
+func agentsByIDExpiresMetric(val float32) fakemetrics.MetricItem {
```

Comment:
nit: `agentsByExpiresAtMetric` for consistency

Reply:
Fixed :)